app/main/data.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from flask import current_app
+
+
+def get_remote_data(hostname, endpoint):
+    """
+        Queries the api endpoint provided and returns a
+        data response in json format.
+
+    Args:
+        endpoint (str): an API get data address
+
+    Returns:
+        data (json): data response in json format
```

Comment:
probabaly should update this to mention it also returns status code, and that it could also return None